_includes/code_snippet.html

```diff
@@ -0,0 +1,23 @@
+{% assign code = include.code %}
+<style>
+    #copyButton {
+        background: transparent;
+        border: none;
+    }
+</style>
+
+<code>{{code}}</code>
+<button
+    id="copyButton"
+    aria-label="Copy to clipboard"
+    onclick="navigator.clipboard.writeText('{{code}}')"
+>
+    📋
+</button>
+
+<script>
+    if(!navigator.clipboard) {
+        document.getElementById("copyButton").remove();
```

Comment:
This won't work if JavaScript is disabled. How having a hidden element with a given `class` which JS is run to make unhidden and add the onclick event to?

Reply:
Done!